### ISSUE TITLE
Shoulder exam: "Speed" test

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10825]
**Preferred term label:**
Shoulder exam: "Speed" test

**Synonyms**

**Definition (free text, please give PubMed ID)**
Manual downward pressure on the arm with the shoulders flexed forward to 90 degrees, elbows fully extended, and palm facing up recreates pain in the anterior shoulder region; named after Dr. Speed

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Biceps tendonitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001150
- label: Positive Speed's test
- parents: [Abnormal shoulder physiology (HP:0033129)](https://hpo.jax.org/app/browse/term/HP:0033129)
- pmid: PMID:9848587
- def: Manual downward pressure on the arm with the shoulders flexed forward to 90 degrees, elbows fully extended, and palm facing up recreates pain in the anterior shoulder region; named after Dr. Speed.
- comment: The Speed's test was recorded as positive if the patient experienced pain in the proximal portion of the shoulder during the application of a downward force applied to the arm. The patient's arm was placed with the forearm in full supination at 90 degrees of shoulder elevation. See Figure 1 of PMID:9848587.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10825
- synonyms:
